libr/core/cmd.c

```diff
@@ -1456,14 +1458,13 @@ static int cmd_kuery(void *data, const char *input) {
 			}
 			cur_cmd = r_str_ndup (out, cur_pos - out);
 
-			r_cons_printf ("\n\n\"%s\" : [", cur_cmd);
+			pj_s (pj, cur_cmd);
 
 			next_cmd = r_str_newf ("anal/%s/*", cur_cmd);
 			temp_storage = sdb_querys (s, NULL, 0, next_cmd);
 
 			if (!temp_storage) {
-				r_cons_println ("\nEMPTY\n");
-				r_cons_printf ("],\n\n");
+				pj_s (pj,"[]");
```

Comment:
remove this line. the pj_ka/pj_end before/after the loop will take care to construct this empty array